test/btwxt_test.cpp

```diff
@@ -199,3 +199,29 @@ TEST_F(TwoDFixture, normalize) {
   EXPECT_THAT(result, testing::ElementsAre(testing::DoubleEq(1.0), testing::DoubleEq(8.832)));
   Btwxt::LOG_LEVEL = 1;
 }
+
+TEST_F(TwoDSimpleNormalizationFixture, normalization_return_scalar) {
+    double return_scalar;
+    std::vector<double> results;
+    double expected_divisor {6.0};
+    double expected_value_at_target {3.5};
```

Comment:
Let's explicitly define this to be what we expect `(x*y/(x_n*y_n))` where in our case `x, y = target = {7, 3}` and `x_n, y_n = normalization_target = {2, 3}`. You can do the same with the `expected_divisor`.

Reply:
These changes should be implemented.

Reply:
This has been implemented. Thanks for guiding me through the process.